src/main/kotlin/build/buf/intellij/module/cache/ModuleCacheV2.kt

```diff
@@ -114,17 +113,12 @@ internal class ModuleCacheV2(env: Map<String, String> = System.getenv()) : BaseM
 
     private fun manifestForModuleKey(key: ModuleKey): Result<Manifest> {
         val bufModuleCachePath = "$bufCacheDir/v2/module/${key.moduleFullName}"
-        val fs = LocalFileSystem.getInstance()
```

Comment:
Migrated this code off using the VFS stuff as if a file isn't cached, we won't find it on disk. Switched to just use standard Java I/O or NIO to handle copying files over from v2 CAS cache to tree based layout.